ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -705,6 +1163,47 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-gcp-ipi-xpn
+- as: gcp-ipi-xpn-mini-perm-arm-f28
```

Comment:
When adding this one, `gcp-ipi-xpn-fips-amd` should be removed, they are duplicated.

Reply:
how about `fips` here? are they different?

Reply:
Similar with cpu-arch case metioned https://github.com/openshift/release/pull/54740#discussion_r1699324320, we are not combing each options, then enumerate all possible different config. For `FIPS` coverage, in e2e, we are also using cross-cover way to test FIPS, select some install profiles to cover fips enabled install, the left ones keep covering non-fips install. For upgrade ci, I am okay with ether of fips or non-fips one, because from the view of the whole upgrade ci, `fips` always can be covered by others.

Reply:
I will remove `gcp-ipi-xpn-fips-amd` since we already have `gcp-ipi-oidc-rt-fips-amd-f28` covered `fips`.